src/stdlib_stats_distribution_PRNG.fypp

```diff
@@ -51,6 +50,8 @@ module stdlib_stats_distribution_PRNG
         integer :: k
 
         k = MAX_INT_BIT_SIZE - bit_size(n)
+        if(k < 0) call error_stop("Error: Integer bit size is greater than 64'  &
+                  //"  bit"
```

Comment:
Could you add the procedure name, i.e. `"ERROR (dist_rand): ..."`, in the error string?

Reply:
OK